test-data/unit/deps.test

```diff
@@ -603,3 +603,11 @@ from a import *
 x = 0
 [out]
 <a[wildcard]> -> m
+
+[case testMissingModuleClass]
```

Comment:
Try with an import from a package as well (such as `from p.b import A`) and using `from p import *`.